src/app/dialogs/drawer-page/drawer-page.component.html

```diff
@@ -162,8 +162,8 @@ <h1>Attack Type: Malware</h1>
   </ngx-section>
 
   <ngx-section class="shadow" sectionTitle="Adding drawer to parent component instead of root">
-    <button type="button" (click)="openDrawer('bottom', undefined, false, undefined, false, parentContainer)">open drawer time</button>
+    <button type="button" (click)="openDrawer('left', undefined, true, editTmpl2, false, parentContainer)">open drawer time</button>
       <div [style.width.%]="100" [style.height.px]="500" [style.background-color]="'red'">
-        <div [style.height.%]="100" #parentContainer></div>
+        <div [style.height.%]="100" [style.position]="'relative'" #parentContainer></div>
```

Comment:
why are we using variable binding syntax with static values here? Instead of doing this you can create classes in the scss

Reply:
none of the pages have scss files but you right, doesnt mean cant take 15 secs to add one ;) added and moved styles